src/main/java/com/geekbeast/rhizome/pods/hazelcast/BaseHazelcastInstanceConfigurationPod.java

```diff
@@ -46,25 +48,46 @@ public Config getHazelcastConfiguration() {
         return config;
     }
 
-    protected NetworkConfig getNetworkConfig( HazelcastConfiguration hzConfiguration ) {
+    @Bean
```

Comment:
Shouldn't be exposed as a bean as changing after hazelcast is created will have no effect.